glusterd2/commands/volumes/volume-reset.go

```diff
@@ -60,6 +60,14 @@ func volumeResetHandler(w http.ResponseWriter, r *http.Request) {
 	oldvolinfo := volinfo
 	// Delete the option after checking for volopt flags
 	opReset := false
+
+	if req.All {
+		req.Options = []string{}
+		for key := range volinfo.Options {
+			req.Options = append(req.Options, key)
```

Comment:
Volume reset all fails if volinfo.Options contains NeverReset/ForceRequired options. Refer https://github.com/gluster/glusterd2/blob/master/glusterd2/commands/volumes/volume-reset.go#L72 and https://github.com/gluster/glusterd2/blob/master/glusterd2/commands/volumes/volume-reset.go#L77

Reply:
@aravindavk  Should the volume reset with force flag pass in case of options that have been set with a force ?

Reply:
You can include only the options with !ForceEnabled and !NeverReset

Reply:
> Should the volume reset with force flag pass in case of options that have been set with a force ?

Looks good. Only exclude NeverReset options